sbt

```diff
@@ -505,6 +545,9 @@ if [[ -n "$noshare" ]]; then
   for opt in ${noshare_opts}; do
     addJava "$opt"
   done
+  if [[ -z "$COURSIER_CACHE" ]]; then
+    export COURSIER_CACHE="$(pwd)/project/.coursier-cache"
```

Comment:
Does this need to be absolute?  The rest of the paths here are relative and we only use `pwd` once for a log message.  So if possible I'd prefer to not use pwd and use a relative path.

Reply:
I had made it absolute just in case, but I guess it should work fine if it isn't.